src/modules/calculatingBullsAndCows.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+const calculateBullsAndCows = (computerDigits, userDigits) => {
+  let bulls = 0;
+  let cows = 0;
+
+  for (let i = 0; i < computerDigits.length; i++) {
+    if (computerDigits[i] === userDigits[i]) {
+      bulls++;
+    }
+
+    if (computerDigits.includes(userDigits[i])) {
+      cows++;
+    }
```

Comment:
If you already did `bulls++` you should not do `cows++`

<img width="833" alt="image" src="https://github.com/mate-academy/node_bulls-and-cows/assets/86789161/813b6cfe-19f9-466e-a767-21b3b85e8a55">